README.md

```diff
@@ -270,6 +274,36 @@ file. As an example
 
 
 
+### pgenv build-dev
+
+This command fetches the current source tree from the official PostgreSQL git repository
+and builds it as special version `dev`. This is an option aimed to help developers
+to keep a version up-to-date with the current PostgreSQL source tree.
+
+Actually, the command works as an *alias* for `build dev`, 
+meaning that the following two commands behave identically:
+    
+    $ pgenv build-dev
+    $ pgenv build dev
+    
+The first time the command is called a source directory `dev` is created and the
+repository is cloned. On a further execution, the command
+will fetch updates from the repository.
+
+It is possible to configure both the repository and the branch or checkout to fetch
+using the variables:
+- `PGENV_POSTGRESQL_REPOSITORY` which defaults to `https://git.postgresql.org/git/postgresql.git`;
+- `PGENV_POSTGRESQL_REPOSITORY_CHECKOUT` which if not set defaults to fetching the `master` branch, while if set creates a tracking branch.
```

Comment:
You like long variable names, don't you? :-D I suggest `_BRANCH` rather than `_CHECKOUT`, since that seems to be a pretty common pattern in my (admittedly limited) experience.

Reply:
Well, I don't like long variable names but I like variable to have explainatory names...which leads to be pretty long...ehm...
By the way, I change to `branch` postfix, even if in my idea  acheckout was intended to be either a commit or a branch.